# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../magicolor2430DL/trivialities.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Some obvious first look / build fixes.

  - Rene Rebe <rene@exactcode.de>

--- magicolor2430DL-1.1.0/Makefile	2004-09-18 02:33:59.000000000 +0200
+++ magicolor2430DL-1.1.0-fixed/Makefile	2005-02-23 08:34:34.284308864 +0100
@@ -2,8 +2,8 @@
 CFLAGS	=	$(RC_CFLAGS)  -I. -I../include
 OPTIM	=	-Wall -fPIC -O2
 #OPTIM	=	-g
-LDLIBS	=	-lcups -lcupsimage
-LDADD	=	/usr/lib/liblcms.a /usr/local/lib/libjbig.a
+LDLIBS	=	-lcups -lcupsimage -llcms -ljbig
+LDADD	=	
 
 MYPRINTER = /usr/local/KONICA_MINOLTA/mc2430DL
 
diff -ur magicolor2430DL-1.1.0/kmlf.h magicolor2430DL-1.1.0-fixed/kmlf.h
--- magicolor2430DL-1.1.0/kmlf.h	2004-09-02 20:18:38.000000000 +0200
+++ magicolor2430DL-1.1.0-fixed/kmlf.h	2005-02-23 09:04:44.613097056 +0100
@@ -33,7 +33,7 @@
 }IMAGEHEADER, *PIMAGEHEADER;
 #pragma pack()
 
-#define arch_is_big_endian 0
+#define arch_is_big_endian __BIG_ENDIAN__
 #define PRINTER_MODEL	"mc2430DL"
 
 #ifdef __linux
